Dockerfile

```diff
@@ -18,12 +18,11 @@ RUN pip3 install -r requirements.txt && pip install uwsgi
 
 ENV DJANGO_SETTINGS_MODULE=core.settings.label_studio
 ENV LABEL_STUDIO_BASE_DATA_DIR=/label-studio/data
-ENV LABEL_STUDIO_PORT="8080"
 
 COPY . /label-studio
 RUN python3.8 setup.py develop
 
-EXPOSE ${LABEL_STUDIO_PORT}
+EXPOSE 8080
```

Comment:
Is this still needed? Does it do anything?

Reply:
I guess it should be there in case you run docker outside heroku environment where expose step is ignored. e.g. by launching it with [readme commands](https://github.com/heartexlabs/label-studio#install-locally-with-docker)